src/Application.php

```diff
@@ -12,9 +12,9 @@
 
 // define STDIN, STDOUT and STDERR if the PHP SAPI did not define them (e.g. creating console application in web env)
 // http://php.net/manual/en/features.commandline.io-streams.php
-defined('STDIN') or define('STDIN', fopen('php://stdin', 'r'));
-defined('STDOUT') or define('STDOUT', fopen('php://stdout', 'w'));
-defined('STDERR') or define('STDERR', fopen('php://stderr', 'w'));
+\defined('STDIN') or \define('STDIN', fopen('php://stdin', 'r'));
+\defined('STDOUT') or \define('STDOUT', fopen('php://stdout', 'w'));
+\defined('STDERR') or \define('STDERR', fopen('php://stderr', 'w'));
```

Comment:
if we're going full \, shouldn't `fopen()` also be backslashed?

Reply:
No. That's low-level opcode stuff.  https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/